components/ww3/cime_config/config_compsets.xml

```diff
@@ -11,37 +11,37 @@
 
   <compset>
     <alias>VWW3-IAF</alias>
-    <lname>2000_DATM%IAF_SLND_SICE_SOCN_SROF_SGLC_WW3</lname>
+    <lname>2000_DATM%IAF_SLND_SICE_SOCN_SROF_SGLC_WW3%sp36x36</lname>
```

Comment:
The `%sp36x36` may not be needed in these aliases since `<value compset="_WW3">sp36x36</value>` is set in `components/ww3/cime_config/config_component.xml`, but I also think it's fine to leave it for clarity's sake.

Reply:
I prefer leave it for clarity.